tutorial/guias/Paso_a_paso_DIA_5.md

```diff
@@ -0,0 +1,192 @@
+
+Como ya dijimos, vamos a ver esto en práctica en la próxima guía. Por ahora volvamos a nuestra clase `Renderer`.
+
+### Función `render_score`
+
+La primera función es `render_score`, que se encarga de representar nuestra puntuación durante el juego. Esta clase recibe varios parámetros:
+
+1. `score`: De tipo entero. Representa la cantidad de puntos obtenida.
+2. `game_window`: No especificamos tipo. Esto será un objeto que nos da Pygame que contiene la ventana de nuestro juego.
+3. `color`: De tipo `Color`. El color con que dibujaremos el score.
+4. `font`: De tipo string. La fuente con la que dibujaremos el score.
+5. `font_size`: De tipo entero. El tamaño de letra con la que se representa el score.
+
+El comentario nos dice que el score lo representaremos en la esquina superior izquierda y nos describe cada argumento de la función.
+
+Luego creamos un objeto `Font` usando el módulo `font` de Pygame. La clase `SysFont` de este módulo recibe la fuente y el tamaño de fuente y devuelve un objeto de tipo `Font`. Luego este objeto que se almacena en la variable `score_font`, lo usamos para crear la superficie del texto que vamos a escribir. El método `render` definido en la clase `Font` recibe el texto a representar, un parámetro de tipo *bool* que determina si queremos que el texto tenga bordes suaves y el color del texto. El llamado a esta función nos devuelve un objeto de tipo `Surface` que representa una región en nuestra pantalla que contiene el texto especificado.
+
+>**Nota :pen:** En la línea anterior usamos lo que se conoce como *string interpolation*. Cuando anteponemos la letra `f` antes de un string en Python, podemos incluir dentro del string código Python. En este caso queremos escribir *Score: x* donde *x* es el número que representa el score. Como dicho score está en la variable `score`, podemos escribir el string como: `f"Score: {score}"`. Al anteponer una `f` y encerrar la variable score entre llaves, obtenemos el resultado deseado.
+>**Nota :pen:** Toda la documentación sobre Pygame la puedes encontrar [aqui](https://www.pygame.org/docs/) (en inglés).
```

Comment:
This note is formatted a little bit weirdly, I think. Should it be a blank line between both notes? I don't know. Take a look at how it looks in the preview and consider adding a blank line or something so the second note is more noticeable.

Reply:
True. I removed the blank line because VS Code was complaining about it, but it's best to keep it